src/utilities/layer-from-json.ts

```diff
@@ -0,0 +1,22 @@
+import * as layer from '../layer'
```

Comment:
Missing semi's here, I think you forgot to run `npm run lint`

Reply:
That is right